expertise/models/specter2_scincl/scincl.py

```diff
@@ -264,7 +265,15 @@ def load_emb_file(emb_file):
                         train_paper_idx.append(paper_id2train_idx[paper_id])
                 train_paper_aff_j = p2p_aff_norm[:, train_paper_idx]
 
-                if self.average_score:
+                if self.percentile_select is not None:
+                    # Select score based on percentile
+                    # q=1.0 (percentile_select=100) -> max score
+                    # q=0.5 (percentile_select=50) -> median score
+                    # q=0.0 (percentile_select=0) -> min score
+                    q = self.percentile_select / 100.0
+                    q = max(0.0, min(1.0, q)) 
+                    all_paper_aff = torch.quantile(train_paper_aff_j, q, dim=1, interpolation='linear')
+                elif self.average_score:
```

Comment:
if we can cover average_score and max_score with percentile_select, do we need to keep them? can we deprecate them?

What about calling it "percentile_score" or "percentile_select_score"?

Reply:
To make sure I understand.
If q=1.0 then, we only take the highest score.
if q=0.5 then, we grab the top 50% scores and average them.
if q=0.0 then, we average all scores.

Is this correct?

Reply:
I asked the chat and it said that it just grabs one score, there is no averaging as I had understood.

> if we can cover average_score and max_score with percentile_select, do we need to keep them? can we deprecate them?

If I understood correctly. Average score wouldn't be replaced by percentiles. @haroldrubio can you confirm 😬 ?

Reply:
Yes there's no averaging - I clarified with Nihar that this is the intended behavior of this feature. So we should keep averaging, but max can be achieved by setting the percentile to 1

Reply:
So if a reviewer has scores: 1, 0.8, 0.4, 0.1 for a particular paper and q=0.75, then, the score that would be used is 0.8. Is this correct?

Reply:
Yes that's correct